Gazelle fails on Windows with "panic: runtime error: invalid memory address or nil pointer dereference"

`bazel run gazelle` fails on Windows with the following error:
> INFO: Running command line: bazel-bin/external/bp_bazel/tools/gazelle/gazelle.exe
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x58 pc=0x9efb76]
goroutine 1 [running]:
github.com/bazel-contrib/rules_jvm/java/gazelle/javaconfig.(*Config).NewChild(...)
        external/contrib_rules_jvm/java/gazelle/javaconfig/config.go:59
github.com/bazel-contrib/rules_jvm/java/gazelle.(*Configurer).Configure(0xc0002c0750, 0xc000221080, {0xc0004e9ad0, 0x29}, 0x0)
        external/contrib_rules_jvm/java/gazelle/configure.go:84 +0x316
github.com/bazelbuild/bazel-gazelle/walk.configure({0xc0002adcc0, 0xa, 0x29?}, 0xc000056500?, 0xc000220fc0?, {0xc0004e9ad0, 0x29}, 0x0)
        external/bazel_gazelle/walk/walk.go:289 +0x383
github.com/bazelbuild/bazel-gazelle/walk.Walk.func1(0xc000220fc0, {0xc000056500, 0x55}, {0xc0004e9ad0, 0x29}, 0x1)
        external/bazel_gazelle/walk/walk.go:145 +0x2d6
github.com/bazelbuild/bazel-gazelle/walk.Walk.func1(0xc000220f00, {0xc0001bb600, 0x4e}, {0xc0004e9230, 0x22}, 0x1)
        external/bazel_gazelle/walk/walk.go:169 +0x8b7
github.com/bazelbuild/bazel-gazelle/walk.Walk.func1(0xc000220300, {0xc0003e5f80, 0x39}, {0xc0004f4c10, 0xd}, 0x1)
        external/bazel_gazelle/walk/walk.go:169 +0x8b7
github.com/bazelbuild/bazel-gazelle/walk.Walk.func1(0xc000220300, {0xc0001ae8d0, 0x2b}, {0x0, 0x0}, 0x0)
        external/bazel_gazelle/walk/walk.go:169 +0x8b7
github.com/bazelbuild/bazel-gazelle/walk.Walk(0xc000220300, {0xc0002adcc0?, 0xa, 0xa}, {0xc0001879f0, 0x1, 0x1}, 0x0, 0xc000251648)
        external/bazel_gazelle/walk/walk.go:179 +0x231
main.runFixUpdate({0xc0001ae690, 0x2b}, 0xc000182080?, {0xc000184210, 0x0, 0x0})
        external/bazel_gazelle/cmd/gazelle/fix-update.go:325 +0xdc5
main.run({0xc0001ae690?, 0x2b?}, {0xc000184210?, 0x1?, 0x1?})
        external/bazel_gazelle/cmd/gazelle/gazelle.go:95 +0x1dd
main.main()
        external/bazel_gazelle/cmd/gazelle/gazelle.go:72 +0xf5

This is because pythonconfig.go#ParentForPackage delegates to filepath.Dir(pkg) which replaces the separator character "" with "/", hence for a path with a 2-levels deep directory, e.g. rootDir\nestedDir1\nestedDir2, a parent with "/" is searched but Configs contains a key with "".

--- a/java/gazelle/javaconfig/config.go
+++ b/java/gazelle/javaconfig/config.go
@@ -2,6 +2,7 @@ package javaconfig
 
 import (
 	"fmt"
+	"path"
 	"path/filepath"
 	"strings"
 
@@ -78,7 +79,7 @@ func (c *Config) NewChild() *Config {
 
 // ParentForPackage returns the parent Config for the given Bazel package.
 func (c *Configs) ParentForPackage(pkg string) *Config {
-	dir := filepath.Dir(pkg)
+	dir := path.Dir(pkg)
 	if dir == "." {
 		dir = ""
 	}

